cli/module_loader.rs

```diff
@@ -248,6 +248,9 @@ impl InFlightModuleLoadsTracker {
       let parsed_source_cache = parsed_source_cache.clone();
       let timeout = self.cleanup_task_timeout;
       let task_handle = tokio::spawn(async move {
+        // NB(bartlomieju): we use a timeout here, which is defined to 10s,
```

Comment:
Nitpick: I feel like we don't have to add attribution to comments like this. It's already in the git blame.

Reply:
I removed it, but it only works as long as this file not moved between files. Then it becomes easy to have a really hard time tracking who wrote the comment.